portfolio/src/main/webapp/index.html

```diff
@@ -35,6 +35,13 @@ <h1>Sena's Portfolio</h1>
         <br/><br/>
     </form>
     <ul id = "commentSection"></ul>
+    <p>Delete comment: </p>
```

Comment:
Love that functionality! One thing you can do to make it easier (so that users don't have to copy and paste the userID) is have a button on each comment to delete it. So you store the ID as part of the comment, then have a button that hits the /deleteComment?idInput={{id}} endpoint when you click on it. This is by no means required though, just something to try if you wanted to try something new :)